x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/BertTokenization.java

```diff
@@ -21,6 +21,8 @@ public class BertTokenization extends Tokenization {
 
     public static final ParseField NAME = new ParseField("bert");
 
+    public static final String MASK_TOKEN = "[MASK]";
```

Comment:
One thing that would be good to do to avoid duplication of constants that must be identical is to use the new constants to set the corresponding ones in the `NlpTokenizer` sub-classes.

For example, in `BertTokenizer.java` change line 42 to `public static final String MASK_TOKEN = BertTokenization.MASK_TOKEN;`.

That way there's no danger of the two constants ever being accidentally updated to be different.

It has to be that way around, because the `ml` plugin depends on the `core` plugin, so `ml` can use things from `core` but not the other way around.